src/styles/main.scss

```diff
@@ -1,3 +1,12 @@
+@import "./utils/mixins";
+@import "./utils/variables";
+
+@import "./blocks/calendar.scss";
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  line-height: $day-size - 2 * $gap-size;
+  text-align: center;
```

Comment:
set this to `&__day`. it's better to vertically cetner text with flex, not line-height